PocketBeagle/Grove/Captouch.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2020 SeeedStudio
```

Comment:
Should be in the same directory as the examples and the examples should use this module.